### ISSUE TITLE
ava 0.15.1 - Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by grahamb** [avajs/babel-plugin-ava-throws-helper#4]
I just upgraded ava from 0.14.0 to 0.15.1, and now one of my [tests](https://github.com/sfu/student-dashboard/blob/master/server/pgt-store/__test__/redis-test.js#L5) is throwing a `TypeError`:

```
test('Throws an error when no redis client is provided', t => {
  t.throws(() => {
    new RedisStore()
  })
})
```

```
NODE_ENV=test ava -v -s

Uncaught Exception: server/pgt-store/__test__/redis-test.js
  TypeError: /Users/grahamb/sfuvault/source/student-dashboard/server/pgt-store/__test__/redis-test.js: Cannot read property 'start' of undefined
    at Object.assertionVisitor.CallExpression (/Users/grahamb/sfuvault/source/student-dashboard/node_modules/babel-plugin-ava-throws-helper/index.js:41:36)
    at NodePath._call (/Users/grahamb/sfuvault/source/student-dashboard/node_modules/babel-traverse/lib/path/context.js:78:18)
    at NodePath.call (/Users/grahamb/sfuvault/source/student-dashboard/node_modules/babel-traverse/lib/path/context.js:49:17)
    at NodePath.visit (/Users/grahamb/sfuvault/source/student-dashboard/node_modules/babel-traverse/lib/path/context.js:108:12)
    at TraversalContext.visitQueue (/Users/grahamb/sfuvault/source/student-dashboard/node_modules/babel-traverse/lib/context.js:174:16)
```

My ava config:

```
"ava": {
    "files": [
      "**/__test__/*.js"
    ],
    "require": [
      "./environment.js",
      "babel-register"
    ],
    "babel": "inherit"
  }
```

Rolling back to 0.14.0 fixes the issue.

**Comment by jamestalmage** [comment]
Can you try swapping out the `no-location-data-fix` branch and see if it works?

**Comment by grahamb** [comment]
👍  It's working with the `no-location-fix` branch & ava 0.15.1.
